### PR TITLE
NbExec | Add S3 configurations to helm

### PR DESCRIPTION
**Pull request by MuthuRajan-S**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Since [S3 buckets are added](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/771882?_a=files) for nbexecution service, and its [added as mandatory helm variables](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/769716?_a=files). This PR is adding this new breaking change in getting started.

### Why should this Pull Request be merged?

Adding a new breaking change made in NbExec helm.

### What testing has been done?

NA

### CODE REVIEW

File: getting-started/templates/systemlink-values.yaml [hunk 1]
```diff
@@ -1008,6 +1008,35 @@ argoworkflows:
 ##
 nbexecservice:
   maxNumberOfWorkflowsToSchedule: *workflowParallelism
+  ## Configure S3/MinIO access.
+  ##
+  s3:
+    ## Secret name for S3 credentials.
+    ##
+    secretName: "executions-s3-credentials"
```
Comment (CiprianAnton) [inline getting-started/templates/systemlink-values.yaml:1016]: I suggest using the default: `nbexecservice-s3-credentials`

File: getting-started/templates/systemlink-values.yaml [hunk 2]
```diff
@@ -1008,6 +1008,35 @@ argoworkflows:
...
+    ## S3 access type. Supported values are "ACCESS_KEY" and "AWS_WEB_IDENTITY_TOKEN".
+    ##
+    authType: "ACCESS_KEY"
```
Comment (CiprianAnton) [inline getting-started/templates/systemlink-values.yaml:1019]: Should not be necessary, should be preconfigured with this. I'm inclined to keep consistency with feedservice

File: getting-started/templates/systemlink-values.yaml (hunk 1 above)
Comment (MuthuRajan-S) [reply getting-started/templates/systemlink-values.yaml]: Updated

File: getting-started/templates/systemlink-values.yaml (hunk 2 above)
Comment (MuthuRajan-S) [reply getting-started/templates/systemlink-values.yaml]: Removed

**Review by prestwick — APPROVED** [review]

**Pull request merged**